src/Content/Text/BBCode.php

```diff
@@ -38,7 +38,7 @@ class BBCode extends BaseObject
 	 *
 	 * @param string $body Message body
 	 * @return array
-	 *                     'type' -> Message type ("link", "video", "photo")
+	 *                     'type' -> Message type ('link', 'video', 'photo')
```

Comment:
This replacement wasn't warranted, double quotes are more suitable for natural language quoting.

Reply:
How is our rule concerning this? By now I'm using ```"``` with SQL conditions and whenever the ```'``` is part of a string.

Reply:
Yes, but for me comments are excluded from this rule as natural language conventions should apply.

Reply:
Can we write down these rules? I have the feeling as if several coders are having different opinions.

Reply:
Is it that important for comments?

Reply:
See https://github.com/friendica/friendica/pull/7746